### ISSUE TITLE
foggy Cityscapes preparation

### ISSUE THREAD

**Issue by xingshuohan** [VisionLearningGroup/DA_Detection#8]

**Comment by xingshuohan** [comment]
Could you show us the details about foggy Cityscapes preparation? Is that same as the GoogleDrive mentioned in the link (https://github.com/tiancity-NJU/da-faster-rcnn-PyTorch)?

**Comment by xingshuohan** [comment]
I got it.

**Comment by benedictflorance** [comment]
Hey @kadebudongtan , I'm getting "Warning: NaN or Inf found in input tensor." and loss becomes NaN after one to two epochs while training from Cityscapes to Foggy Cityscapes. Did you experience this?

**Comment by AmingWu** [comment]
Hey, @benedictflorance, I also meet the question of Nan, Have you solved this problem?
